### PR TITLE
ArchivesSpaceClient: offer inherit_notes opt to add_digital_object

### PR DESCRIPTION
**Pull request by sevein**

### CODE REVIEW

**Comment by sevein** [comment]
Ready!

File: misc/Dockerfile.2 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM python:2
```
Comment (Hwesta) [inline misc/Dockerfile.2]: Are you planning on committing these? If so, it should probably be its own Docker-related PR.

**Comment by Hwesta** [comment]
:+1:  on everything except the Docker changes, see comment.

File: misc/Dockerfile.2 (hunk 1 above)
Comment (sevein) [inline misc/Dockerfile.2]: I'll delete it. It was an example I wanted to show you.

**Pull request closed without merging**